CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## 0.5.3dev
+* [Feature] Adds telemetry collect for `%sqlplot` invocation ([#89](https://github.com/ploomber/jupysql/pull/89))
```

Comment:
Probably a Fix and not a feature

Reply:
[5ddf7c6](https://github.com/ploomber/jupysql/pull/89/commits/5ddf7c6d5e15dac6008c8de5000a2350950e1833)